modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -30,11 +30,42 @@ Java 11 has various enhancements like lambda local variable type inference, prof
     <dependency>
         <groupId>com.couchbase.client</groupId>
         <artifactId>java-client</artifactId>
-        <version>3.2.3</version>
+        <version>3.2.4</version>
     </dependency>
 </dependencies>
 ----
 
+== Version 3.2.4 (9 December 2021)
+
+Version 3.2.3 is the fifth release of the 3.2 series.
+
+http://packages.couchbase.com/clients/java/3.2.4/Couchbase-Java-Client-3.2.4.zip[Download] |
+https://docs.couchbase.com/sdk-api/couchbase-java-client-3.2.4/index.html[API Reference] |
+http://docs.couchbase.com/sdk-api/couchbase-core-io-2.2.4/[Core API Reference]
+
+The supported and tested dependencies for this release are:
+
+* com.couchbase.client:**java-client:3.2.4**
+* com.couchbase.client:**core-io:2.2.4**
+* io.projectreactor:**reactor-core:3.4.12**
+* org.reactivestreams:**reactive-streams:1.0.3**
+
+Optional artifacts on top of this SDK version are tested for the following compatibilities:
+
+.Optional Artifact Version Compatibility
+[options="header"]
+|=======================
+| Artifact                  | Version | Built Against             | API Stability
+| `tracing-opentelemetry`   | 1.0.4   | OpenTelemetry 1.7.1       | Committed
+| `tracing-opentracing`     | 1.0.4   | OpenTracing 0.33.0        | Committed
+| `metrics-opentelemetry`   | 0.2.4   | OpenTelemetry 1.7.1-alpha | Volatile
+| `metrics-micrometer`      | 0.2.4   | Micrometer 1.7.5          | Volatile
+|=======================
+
+=== Improvements
+* https://issues.couchbase.com/browse/JCBC-1872[JCBC-1872]:
```

Comment:
This is also in the release notes for 3.2.2, is this right @daschl?

Reply:
wait, I messed this up